src/components/Image/Image.stories.tsx

```diff
@@ -10,9 +13,9 @@ const meta = {
   },
   argTypes: {
     ratio: {
-      options: ['1:1', '4:3', '16:9', '21:9'],
+      options: [...IMAGE_RATIO_OPTIONS],
       table: {
-        type: { summary: `"1:1", "4:3", "16:9", "21:9"` }
+        type: { summary: ratioOptionsString }
```

Comment:
Heads up that this didn't include the **50:50** ratio that was available in the type on line 23 of `components/Image/Image.tsx` and is included here. Let me know if it should be removed.

Reply:
Let's exclude 50:50 on image since it's only intended for use in the product card layout. We can revisit this when we refactor card in a few sprints.